tests/helpers/perf_run_helper.rb

```diff
@@ -461,25 +461,40 @@ def generate_scale_scenarios(gatling_scenario)
     file = File.read("#{scenarios_dir}/#{gatling_scenario}")
     json = JSON.parse(file)
 
-    # allow the base instances to be set via environment variable
+    # allow the settings to be set via environment variables
+    # TODO: refactor to streamline handling of multiple environment variables / settings
```

Comment:
What is the use case for this?  Is a ticket needed?

Reply:
I was really just noting that it seems like this could be streamlined now that multiple environment variables are being handled here in a similar way. Given that we have the agent simulator on the horizon it doesn't seem to make sense to write (and groom) a ticket for work that may never be done. If we reach a point where we are sticking with GPLT and looking for opportunities for improvement we can generate tickets from the TODOs.